### ISSUE TITLE
About the training time of RIFE

### ISSUE THREAD

**Issue by JinxLv** [hzwer/Practical-RIFE#137]
Hi, hzwer:
This is a really good project for the community of VFI. I have a question about the training time of RIFE: How much time you cost to train RIFE-lite from scratch? 
Since it usually costs me too much training time when changing the structure of the model.  I use a small dataset(~2K pairs image, 540P resolution), and the model usually takes about >=5 days to converge (observing the training curve through tensorboard). My implementation details are: 1e-4 learning rate, 384*384 patch size, 80 batch size, 1 GPU resource. I'm confused about whether it is normal or abnormal? Looking forward to your reply!

**Comment by hzwer** [comment]
Hi, I use 8 gpus * 7+days to train the model. These low-level models typically exhibit a phenomenon where they can be trained for 1000+ epochs and still show performance improvements. So I think instead of waiting for it to converge, it's better to check whether the performance is sufficient for the application scenario.

**Comment by JinxLv** [comment]
OK, Thanks for your valuable advice!
